src/lib/connectors/jsdom/jsdom.ts

```diff
@@ -226,7 +226,7 @@ class JSDOMConnector implements IConnector {
      * * uses `favicon.ico` and the final url after redirects.
      */
     private async getFavicon(element?: HTMLElement) {
-        const href = element ? element.getAttribute('href') : '/favicon.ico';
+        const href = (element && element.getAttribute('href')) || '/favicon.ico';
```

Comment:
Any reason to do assignment in this way instead of the old way?

In terms of readability I prefer the old way.